extension.js

```diff
@@ -34,16 +34,16 @@ const prettyPrint = Utils.prettyPrint;
 const writeRegistry = Utils.writeRegistry;
 const readRegistry = Utils.readRegistry;
 
-let TIMEOUT_MS           = 1000;
-let MAX_REGISTRY_LENGTH  = 15;
-let MAX_ENTRY_LENGTH     = 50;
-let CACHE_ONLY_FAVORITE  = false;
-let DELETE_ENABLED       = true;
-let ENABLE_KEYBINDING    = true;
-let PRIVATEMODE          = false;
-let NOTIFY_ON_COPY       = true;
-let MAX_TOPBAR_LENGTH    = 15;
-let TOPBAR_DISPLAY_MODE  = 1; //0 - only icon, 1 - only clipbord content, 2 - both
+var TIMEOUT_MS           = 1000;
```

Comment:
Why did you change these?

Reply:
In dandoid's log above, there are those lines : 

Some code accessed the property 'Fields' on the module 'prefs'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.

Reply:
Ah, ok. But that warning is referring to the ``Fields`` variable from the ``prefs.js`` file. See it [here](https://github.com/Tudmotu/gnome-shell-extension-clipboard-indicator/blob/master/prefs.js#L12). The warning is actually about the [usage of that variable](https://github.com/Tudmotu/gnome-shell-extension-clipboard-indicator/blob/master/extension.js#L488) from within ``extension.js``.

To fix this warning, just change the ``const Fields = { ...`` in ``perfs.sj`` to ``var Fields = { ...``. Also, to make sure your changes do resolve the issue, you can take a look at the logs using journalctl: ``$ journalctl -f /usr/bin/gnome-shell``.

I am unsure how @danboid reproduced this issue, but I think changing the ``Fields`` variable definition should fix it.

Can you please change these back to ``let``?
Thanks!

Reply:
You know what, never mind. I am testing it now anyway, so I will make the changes.
Again, thank you for this feature and the effort! :smiley: